benchmark/src/main/java/com/datatorrent/benchmark/window/README.md

```diff
@@ -0,0 +1,31 @@
+Windowed Operator Benchmark Readme
+
+Introduction
+The Windowed operator benchmark are applications to test the performance of windowed operator. 
+This benchmark applications only available for Malhar 3.7.0(or 3.7.0-snapshot) or later versions. 
+The source code is located inside the malhar/benchmark. There are two applications, 
+one is benchmark for WindowedOperator, another is for KeyedWindowedOperator. 
+
+Steps of compile
+- Get source code of malhar 3.7.0 or 3.7.0-snapshot
+- Compile the malhar
```

Comment:
Just build with -Pall-modules instead of 3 step process. Provide the build command line here.